### DISCUSSION TITLE
Is it possible to get information from private repos?

### DISCUSSION THREAD

**Discussion by dimkagithub** [badges/shields#9824] [Q&A] [answered]
Hi!
Is it possible to get information from private repos?
I tried to deploy in Vercel but I get error — Page Not Found 404.
I am completely newbie with Vercel :)

**Comment by chris48s** [comment, marked as answer]
Hi.

https://shields.io/ can't access your private repos

If you self-host your own instance, you can give it an auth token that does have the ability to read your private repos: https://github.com/badges/shields/blob/master/doc/server-secrets.md#github

There are some docs on self-hosting https://github.com/badges/shields/blob/master/doc/self-hosting.md

Here's the last thread from someone trying to run on vercel which does contain some info https://github.com/badges/shields/discussions/9521 but I think that user decided to host elsewhere.
